docs/reference/api/connectivity/bluetooth/GattServer.md

```diff
@@ -1,6 +1,31 @@
 ## GattServer
 
-[Add description here.]
+A GattServer is a collection of GattServices. These services contain characteristics that a peer connected to the device may read or write. These characteristics may also emit updates to subscribed clients when their values change.
+
+#### Server layout
+
+Application code can add a GattService object to the server with the help of the function `addService()`. That function registers all the GattCharacteristics enclosed in the service, as well as all the characteristic descriptors (see GattAttribute) that these characteristics contain. Service registration assigns a unique handle to the various attributes that are part of the service, and this handle should be used to subsequently read or write these components.
```

Comment:
Query: Who or what should use this handle to read or write these components?

Reply:
The application. Code written by the user.